standard_scenarios_2021/dsgrid_project/datasets/sector_models/tempo_standard_scenarios_2021/dimensions/weather.csv

```diff
@@ -0,0 +1,2 @@
+id,name
+tempo_weather,"Weather year used by TEMPO."
```

Comment:
... what is that weather year?

Reply:
I copied these files directly from the dsgrid Standard Scenarios project.

Reply:
@mooneyme @elainethale Am I supposed to change something about this?

Reply:
Hopefully it is 2012?